src/zarr/core/array.py

```diff
@@ -152,15 +152,14 @@ async def create(
 
         shape = parse_shapelike(shape)
 
-        if chunk_shape is None:
-            if chunks is None:
-                chunk_shape = chunks = _guess_chunks(shape=shape, typesize=np.dtype(dtype).itemsize)
-            else:
-                chunks = parse_shapelike(chunks)
+        if chunks is not None and chunk_shape is not None:
+            raise ValueError("Only one of chunk_shape or chunks can be provided.")
```

Comment:
I really don't like that we have both of these in the constructor. Seems like we can get away with only `chunks`.

Reply:
I want that to get sorted out elsewhere.